Character sheets fix

marxpark00 opened the pull request:
Issues:
 Add indoctrination counter on to the character sheet #19 
 Vehicle Handling Click to Roll #4 ( I added the vehicle handling as "DND5E.skills['ani'] = 'Vehicle Handling' " in the js of the module)
 ME5e Deafault, Paragon, and Renegade Character Sheets Color schemes are hard to read #5

marxpark00 committed:
[Issue-#5] Fix styles,background image and colours of text for multiples areas in the sheets, i added (DND5E.skills['ani'] = 'Vehicle Handling';) in me5e.js for foundry to detect that skill

marxpark00 committed:
[Issue-#5] Fix styles and background image

marxpark00 committed:
[Issue-#5] Change styles and gray colour

marxpark00 committed:
[Issue-#19] Added Indoctrination input

marxpark00 committed:
[Issue-#5] 💄 Change styles, adding white in tabs and in descriptions

marxpark00 committed:
[Issue-#5] 💄 Change styles in inventory list

marxpark00 committed:
[Issue-#5] 💄 Change styles, deleteing background colors and adding the correct family font

marxpark00 committed:
[Issue-#5] 💄 Change styles, deleteing background colors and adding the correct family font

marxpark00 committed:
[Issue-#5] 💄 Revert border

marxpark00 committed:
[Issue-#5] 💄 Change styles in spell inventory

marxpark00 committed:
[Issue-#5] 💄 Change font family in summary

marxpark00 committed:
[Issue-#5] 💄 Change colour for me5 sheet

bushongj commented:
Hey @marxpark00 the character sheets are not working on my end. Is there some module youre using to make these sheets work? See screen shot. A lot of the values are showing up as NAN.  On youre end could you try creating a DnD5e world with just the ME5e module to see if you can replicate what I'm seeing? 
<img width="711" alt="NANCharacter" src="https://user-images.githubusercontent.com/41301317/158088073-41eef0a0-18e5-48fb-bc13-7e2371c8a681.PNG">

bushongj commented on src/me5e.js:
```diff
@@ -1,174 +1,197 @@
+DND5E.toolProficiencies['navg'] = 'Starship System (Navigation)';
+DND5E.toolProficiencies['pois'] = "Brewer's Supplies";
+DND5E.toolProficiencies['aswb'] = "Armorsmith's Workbench";
+DND5E.toolProficiencies['cook'] = "Cook's Utensils";
```
Please remove ` DND5E.toolProficiencies['cook'] = "Cook's Utensils";`  . This will bring back the cooking utensil issue: https://github.com/Astrotech89/ME5e-FoundryVTT/issues/1

marxpark00 replied:
i did right now, i didnt see that issue sorry

marxpark00 committed:
[Issue-#5] 💄 Change fields in character sheet

marxpark00 committed:
[Issue-#5] delete cooking utensils

bushongj committed:
Update me5e.js

removing reference to shield attribute which is causing the NaN error on character sheets. shields is not something that you can call from the dnd5e game system

bushongj commented:
Commented out refrence to shields attribute from the me5e.js file. That is not an attribute from the DnD5e game system that you can call. Otherwise everything else looks great
